crates/extensions/tedge_health_ext/src/lib.rs

```diff
@@ -13,18 +13,23 @@ use tedge_actors::RuntimeRequestSink;
 use tedge_actors::ServiceConsumer;
 use tedge_actors::ServiceProvider;
 use tedge_actors::SimpleMessageBoxBuilder;
-use tedge_api::health::health_status_down_message;
-use tedge_api::health::health_status_up_message;
+use tedge_api::health::ServiceHealthTopic;
+use tedge_api::mqtt_topics::Channel;
+use tedge_api::mqtt_topics::MqttSchema;
+use tedge_api::mqtt_topics::OperationType;
+use tedge_api::mqtt_topics::Service;
 use tedge_mqtt_ext::MqttConfig;
 use tedge_mqtt_ext::MqttMessage;
 use tedge_mqtt_ext::TopicFilter;
 
 pub struct HealthMonitorBuilder {
-    service_name: String,
+    service_health_topic: ServiceHealthTopic,
     box_builder: SimpleMessageBoxBuilder<MqttMessage, MqttMessage>,
 }
 
 impl HealthMonitorBuilder {
+    /// Creates a HealthMonitorBuilder that creates a HealthMonitorActor with
+    /// old topic scheme.
```

Comment:
Any specific reason why we decided not to update the producers as part of this PR? Just to keep this PR simpler or were there some reasons that forced us to keep them on the old scheme? I'm fine with updating them all in a separate PR. But, just wanted to know if simplicity was the only reason.